librz/analysis/class.c

```diff
@@ -1342,8 +1351,12 @@ static void list_all_functions_at_vtable_offset(RzAnalysis *analysis, const char
 			continue;
 		}
 
-		if (vtableContext.read_addr(analysis, vtable->addr + offset, &func_address))
-			rz_cons_printf("Function address: 0x%08" PFMT64x ", in %s vtable %s\n", func_address, class_name, vtable->id);
+		if (vtableContext.read_addr(analysis, vtable->addr + offset, &func_address)){
+			RzAnalysisFunction *fcn = rz_analysis_get_fcn_in(analysis, func_address, 0);
+			const char *noMethodName = "No Name found";
+			const char *const func_name = fcn ? fcn->name : noMethodName;
+			rz_cons_printf("Function name: %s, address: 0x%08" PFMT64x ", in %s vtable %s\n", func_name, func_address, class_name, vtable->id);
```

Comment:
Better just show the previous `Function address: 0x...` in cases when no function has been found instead of `Function name: No name found` since that is kind of meaningless.